engine/util/util.go

```diff
@@ -276,6 +276,13 @@ func StructKindToFieldNameTypeMap(kind string) (map[string]*types.Type, error) {
 		//	continue
 		//}
 
+		// Skip unexported fields. These should never be mapped golang<->mcl.
```

Comment:
I'll merge this commit from the open PR. Sorry for the delay.

Reply:
No problem. I'll try to remember not squashing it into the final commit along with the other stuff.